lib/js/components/PopOver/PopOver.vue

```diff
@@ -208,6 +209,16 @@ export default {
 			type: String,
 			default: null,
 		},
+		appendToBody: {
+			type: Boolean,
+			default: false,
+		},
+		modifiers: {
+			type: Object,
+			default() {
+				return {};
+			},
```

Comment:
You can use shortcut :) 
```suggestion
			default: () => ({}),
```